src/fiatExchanges/quotes/normalizeQuotes.test.ts

```diff
@@ -38,6 +42,29 @@ describe('normalizeQuotes', () => {
       ['Moonpay', 10],
     ])
   })
+
+  it('sorts FiatConnect quotes with no fee returned at the end of quotes', () => {
```

Comment:
this is fine, but there should also be a test for the higher-level behavior you were trying for with the bugfix